app/scripts/comp/launcher-electron.js

```diff
@@ -244,6 +244,9 @@ const Launcher = {
             }
         }
         return ps;
+    },
+    openFileChooser: function(callback, button) {
```

Comment:
please don't access any UI elements in Launcher. Instead, you can create a property `canChooseFile` and check it in calling code.